Src/FluentAssertions/Equivalency/Field.cs

```diff
@@ -8,7 +8,7 @@ namespace FluentAssertions.Equivalency;
 /// <summary>
 /// A specialized type of <see cref="INode  "/> that represents a field of an object in a structural equivalency assertion.
 /// </summary>
-public class Field : Node, IMember
+internal class Field : Node, IMember
```

Comment:
_Technically_ these are breaking changes. Should we mention this?

Reply:
Yes, see also the comment from @jnyrup.